packages/lib/sdk/src/utils/svelte/buildUrl.js

```diff
@@ -0,0 +1,28 @@
+import { config } from '$evidence/config';
+/**
+ * Adjusts a path to include the configured base path
+ * Ignores undefined, and absolute URLs
+ * @param {string} path 
+ * @example buildUrl('http://localhost:3000/test') // 'http://localhost:3000/test'
+ * @example buildUrl('/test') // '/base/test'
+ * @example buildUrl(undefined) // undefined
+ * 
+ * @returns 
+ */
+export const buildUrl = (path) => {
```

Comment:
I think this could be named better, maybe `addBasePath`? Something that describes what it does specifically